README.md

```diff
@@ -54,6 +46,14 @@ You can install Harbor with the helper script:
 After that you can use docker-compose to run the whole harbor system:
 - `docker-compose -f .harbor/harbor/docker-compose.yml up -d`
 
+Once Harbor is up and running (see the section "Setup Harbor" below), you can
```

Comment:
This will need rewording now, not sure if this change was intentional (seems a bit unrelated).

Might be interesting to keep though a note before too, something like "we will be able to test if that choice works after harbor is up".

Reply:
You are right that this does not really belong here, so I've dropped the last commit and will do a more comprehensive edit of the README separately.